### PR TITLE
[18.0][MIG] hr_timesheet_analytic_tag: Migration to 18.0 

### PR DESCRIPTION
**Pull request by peluko00**
Module migrated to version 18.0

cc https://github.com/APSL 163679
@miquelalzanillas @javierobcn @mpascuall @BernatObrador @ppyczko please review

- [Issue] https://github.com/OCA/account-analytic/issues/696

Depends on:

- [account_analytic_tag] https://github.com/OCA/account-analytic/pull/709

### CODE REVIEW

**Review by ppyczko — APPROVED** [review]
LGTM!

**Review by mpascuall — APPROVED** [review]
LGTM, tested in runboat

**Review by BernatObrador — APPROVED** [review]
LGTM

File: hr_timesheet_analytic_tag/tests/test_hr_timesheet_analytic_tag.py [hunk 1]
```diff
@@ -63,27 +63,23 @@ def _create_hr_timesheet(self):
         )
 
     def test_hr_timesheet_without_tags(self):
-        self.task.analytic_account_id = self.analytic_account_1
```
Comment (pedrobaeza) [inline hr_timesheet_analytic_tag/tests/test_hr_timesheet_analytic_tag.py]: Why removing this?

Comment (peluko00) [reply hr_timesheet_analytic_tag/tests/test_hr_timesheet_analytic_tag.py]: In v18 this field was removed from `project.task` model.

Comment (pedrobaeza) [reply hr_timesheet_analytic_tag/tests/test_hr_timesheet_analytic_tag.py]: Yeah, but then, `analytic_account_1` is never used. If not needed, remove its creation.

Comment (peluko00) [reply hr_timesheet_analytic_tag/tests/test_hr_timesheet_analytic_tag.py]: Done! Thanks!

**Review by pedrobaeza — APPROVED** [review]
/ocabot migration hr_timesheet_analytic_tag
/ocabot merge nobump

**Comment by pedrobaeza** [comment]
@sbidoul the bot seems stuck.

**Pull request merged**